gnocchi/cli.py

```diff
@@ -83,7 +83,16 @@ def change_sack_size():
     ])
     conf = service.prepare_service(conf=conf)
     s = storage.get_driver(conf)
-    report = s.incoming.measures_report(details=False)
+    try:
+        report = s.incoming.measures_report(details=False)
+    except TypeError:
```

Comment:
Good idea, but the implementation is too vague. We need a proper exception caught defined by gnocchi.storage. TypeError could be raised by more or less anything, especially when we'll modify the code.